packages/slice-machine/src/modules/modal/types.ts

```diff
@@ -0,0 +1,10 @@
+export enum ModalKeysEnum {
+  LOGIN = "LOGIN",
+}
+
+export type ModalStoreType = Record<ModalKeysEnum, boolean>;
+
+export type UserContextStoreType = {
```

Comment:
I'm not sure to understand why this is here, it doesn't seems to be modal related.

Reply:
fixed